llvm/lib/Transforms/Scalar/RewriteStatepointsForGC.cpp

```diff
@@ -1514,7 +1514,7 @@ static void CreateGCRelocates(ArrayRef<Value *> LiveVariables,
   auto getGCRelocateDecl = [&](Type *Ty) {
     assert(isHandledGCPointerType(Ty, GC));
     auto AS = Ty->getScalarType()->getPointerAddressSpace();
-    Type *NewTy = Type::getInt8PtrTy(M->getContext(), AS);
+    Type *NewTy = PointerType::getUnqual(M->getContext(), AS);
```

Comment:
Thank you for the update 👍 

The following update is needed to avoid build failure.
```suggestion
    Type *NewTy = PointerType::get(M->getContext(), AS);
```

Reply:
Right - thanks for pointing that out.